website/content/api-docs/system/internal-counters.mdx

```diff
@@ -89,24 +89,89 @@ $ curl \
+mount path within a given namespace to the overall activity of the namespace.
+
+- `months` breakdowns provide attributions of each month to the total activity
+count. These month level attributions are further broken down into namespace and mount
+level attributions for each month.
+
+- Each entry in the `months` breakdown contains a `new_clients` block. Within a
+billing period, when a token is first used, regardless of when it was created,
+that token is considered a new client for that billing period. This means that
+all the active clients in the first month of the billing period will be
+considered new clients for that billing period. These tokens could all be
+generated and counted in the previous billing period, but despite that they are
+still considered new clients in the context of the given billing period. For
+each subsequent month in the same billing period, the tokens used in those
+respective months that were not used in any of the previous months of the same
+billing period are considered new clients for that month. Hence, the computation
+of new clients differs for each combination of `start_time` and `end_time`.
+
+- The `new_clients` block within the `months` breakdown will also be further
+broken down by namespaces and mounts for visibility into which components in
+Vault lead to the new clients for each month.
+
+- The `distinct_entities` field name is deprecated since Vault 1.10. Refer to
+`entity_clients` field instead. The `distinct_entities` field is currently
+returned by the API for backwards compatibility and it may be removed in future.
+
+- The `non_entity_tokens` field name is deprecated since Vault 1.10. Refer to
+`non_entity_clients` field instead. The `non_entity_tokens` field is currently
+returned by the API for backwards compatibility and it may be removed in future.
+
+- All occurrences of namespace breakdowns, mount level breakdowns and month
+level breakdowns in the response are arranged in the decreasing order of their
+activity. Entries with highest activity will be at the top.
+
+- If the `end_date` supplied to the API is for the current month, the activity
+information returned by this API will only be till the previous month. The
+activity system is designed to process the accumulated activity only at the end
+of the month. Since the current month's information is not fully processed by
+the system it will not be added to the API response. However the accumulated
+response can be viewed using the partial month API.
+
+- The response includes the actual time period covered, which may not exactly
+match the query parameters due to the month granularity of data, or missing
+months in the requested time range.
+
+For more information on how clients map to these client IDs, and how clients are
+counted, please visit the [client count](/docs/concepts/client-count) concepts
+page.
+
+A time period may be specified; otherwise it reports on a default reporting
```

Comment:
This duplicates a lot of what precedes it.  You might wish to get rid of it, or maybe move it back to the top as a kind of summary.

Reply:
After trying some rearrangements, decided that its not worth it. Removed it.